lib/validator/validator.go

```diff
@@ -348,6 +349,16 @@ func validateIPPool(v *validator.Validate, structLevel *validator.StructLevel) {
 			}
 		}
 
+		// The Calico CIDR should be strictly masked
+		if !pool.Spec.Disabled {
```

Comment:
I don't think we should check the Disabled flag here - we should prevent configuration or disabled pools too.